lib/real-device.js

```diff
@@ -314,14 +314,15 @@ export class RealDevice {
    * @param {import('./driver').XCUITestDriverOpts} opts
    * @returns {Promise<void>}
    */
-  async reset({bundleId, fullReset, platformVersion}) {
+  async reset({bundleId, fullReset}) {
     if (!bundleId) {
```

Comment:
we may probably combine below conditions using ||

Reply:
makes sense